### ISSUE TITLE
Crash in Quadtastic v0.6.3

### ISSUE THREAD

**Issue by polm** [25A0/Quadtastic#35]
Quadtastic crashed with the following error message:

```
/AppLogic.lua:27: ///QuadExport.lua:29: assertion failed!
stack traceback:
main.lua:324: in function <main.lua:315>
[C]: in function 'assert'
/AppLogic.lua:27: in function 'run'
/AppLogic.lua:123: in function 'pop_state'
/AppLogic.lua:51: in function 'save'
//Dialog.lua:488: in function 'f'
main.lua:226: in function 'draw'
[string boot.lua]:468: in function <[string boot.lua]:436>
[C]: in function 'xpcall'
[string boot.lua]:569: in function <[string boot.lua]:564>
```

I chose "Save", typed a name in the file path, and pushed Enter. 

Besides this issue, for some reason the "save" and "export" buttons are always disabled, so I can't actually save anything. I tried running the program from a terminal but didn't see any relevant output. 

---
Affects: v0.6.3
Edition: windows

**Comment by 25A0** [comment]
Thanks for reporting! I'll look into it.

**Comment by 25A0** [comment]
That should be fixed in #36. The fix will be released in v0.6.4 on Wednesday.

In case you want to test it before that, here is the Windows build:
[Quadtastic.zip](https://github.com/25A0/Quadtastic/files/1425758/Quadtastic.zip)

**Comment by polm** [comment]
Thanks for the quick fix! I can confirm it works.
